Formula/autoenv.rb

```diff
@@ -16,4 +16,11 @@ def caveats; <<-EOS.undent
       source #{opt_prefix}/activate.sh
     EOS
   end
+
+  test do
+    (testpath/"test").mkpath
```

Comment:
per Dom's comment I guess this line isn't actually needed due to the `.write` which creates the directory for you